src/index.js

```diff
@@ -0,0 +1,162 @@
+const tempDisplay = document.getElementById('tempValue');
+const increaseTemp = document.getElementById('increaseTempControl');
+const decreaseTemp = document.getElementById('decreaseTempControl');
+const landscape = document.getElementById('landscape');
+const cityNameDisplay = document.getElementById('headerCityName');
+const cityNameInput = document.getElementById('cityNameInput')
+const tempButton = document.getElementById('currentTempButton');
+const skySelect = document.getElementById('skySelect');
+const gardenContent = document.getElementById('gardenContent');
+const skyDisplay = document.getElementById('sky');
+const resetButton = document.getElementById('cityNameReset');
+const defaultCity = 'Seattle';
+
+let temperature = 88;
+tempDisplay.innerText = temperature;
+
+// Color of the temperature value number
+const updateTempColor = () => {
+tempDisplay.classList.remove('red', 'orange', 'yellow', 'green', 'teal');
+
+if (temperature >= 80) {
+    tempDisplay.classList.add('red');
+} else if (temperature >= 70) {
+    tempDisplay.classList.add('orange');
+} else if (temperature >= 60) {
+    tempDisplay.classList.add('yellow');
+} else if (temperature >= 50) {
+    tempDisplay.classList.add('green');
+} else {
+    tempDisplay.classList.add('teal');
+}
+};
```

Comment:
Be mindful of indentation

```suggestion
const updateTempColor = () => {
  tempDisplay.classList.remove('red', 'orange', 'yellow', 'green', 'teal');

  if (temperature >= 80) {
      tempDisplay.classList.add('red');
  } else if (temperature >= 70) {
      tempDisplay.classList.add('orange');
  } else if (temperature >= 60) {
      tempDisplay.classList.add('yellow');
  } else if (temperature >= 50) {
      tempDisplay.classList.add('green');
  } else {
      tempDisplay.classList.add('teal');
  }
};
```

Reply:
Instead of using if/else if/else which can be brittle and bug-prone, consider using a list to store the landscape emojis. Then you can write logic that compares the first element in each inner list against `temperature ` and grab the color or emojis. 

```js
temperatureColorsAndLandscapes = [
    [80, "red", "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂"],
    [70, "orange", "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷"],
    [60, "yellow", "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃"],
    [50, "green", "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"],
    [40, "teal", "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"],
]
```